src/devtools/client/debugger/src/components/SecondaryPanes/Breakpoints/BreakpointTimelinePoint.js

```diff
@@ -89,7 +64,7 @@ function BreakpointTimelinePoint({
       })}
       title={`${index + 1}/${analysisPoints.length}`}
       onClick={() => seek(point.point, point.time, true)}
-      style={{ left: `${leftPercentOffset}%` }}
+      style={{ left: `calc(${leftPercentOffset}% - ${pointWidth / 2}px)` }}
```

Comment:
I think we could do this even simpler. To get the same half-of-width-displacement effect:
```
.breakpoint-navigation-timeline-point {
    transform: translateX(60px);
}
```
That way JS doesn't ever need to know about `pointWidth` in the first place. Reduces the code here to just `style={{ left: `${leftPercentOffset}%` }}`

Reply:
That was my first idea, but then I saw that the javascript variable is used in 2 other places, so it's probably a good idea to have a single source of truth here.